### ISSUE TITLE
Invalid illuminants

### ISSUE THREAD

**Issue by andyp6** [ampas/rawtoaces#51]
Entering D55 as shown in the example string in the README produces an error:

`$ rawtoaces --wb-method 0 D55 --mat-method 0 input.raw`

```
Error: Unknown light source  -D55.
Please use "--valid-illum" to see list of available light sources.
```

Running --valid-illum produces the following list

```
3200k
cie15-a
cie15-c
cie15-d50
cie15-d55
cie15-d65
cie15-d75
d40
d45
d50
d60
iso7589
```
d50 seems to be a higher precision version of cie15-d50 but is there need for both?

**Comment by aforsythe** [comment]
We're doing some major cleanup here.  @miaoqi is working on functions for daylight and planckian sources.  We'll then only have to read in sources from files like fluorescent lamps, HMI, etc.

**Comment by aforsythe** [comment]
This issue was moved to http://www.acescentral.com/t/invalid-illuminants/1154

**Comment by miaoqi** [comment]
@andyp6 This issue should have been addressed in the current version (0.1.0). Can you close it? Thanks!

**Comment by andyp6** [comment]
Sure. Sorry I forgot to close this.
